tests/cli_ui_test.py

```diff
@@ -102,6 +159,14 @@ def ask(self):
     if question_type == "non_implemented":
         with pytest.raises(NotImplementedError):
             results = input_collector.collect()
+    elif (
+        question_type == "model"
+        and expected_results.get("prompt_ret").get("model_description") == ""
+        or expected_results.get("prompt_ret").get("wants_to_document_model") is False
+    ):
+        print(expected_results)
```

Comment:
is this print suppose to be here?

Reply:
Good catch! I forgot to remove it after debugging the test.